src/api/Controllers/UploadController.cs

```diff
@@ -388,6 +522,71 @@ public CsvImportBoreholeMap()
         }
     }
 
+    private sealed class CsvImportLithologyMap : ClassMap<LithologyImport>
```

Comment:
PP: Die `ClassMap`s sollten wir ab jetzt in eigene Files auslagern, da sonst diese Klasse viel zu gross wird. Vielleicht können wir dann noch z.B. die `CsvConfiguration` auslagern und in allen `ClassMap`s wiederverwenden.

Gehört zum Thema Refactoring und kann in einem separaten PR geschehen.

Reply:
Ok. Wie unten erwähnt, können wir das anschauen.